### DISCUSSION TITLE
Importing YAML and TOML as object

### DISCUSSION THREAD

**Discussion by mbledkowski** [nuxt/nuxt#25721] [Questions]
Hello,
How can I add functionality, that would allow me to import YAML/TOML files?
![image](https://github.com/nuxt/nuxt/assets/23295125/62ffa1e0-2b5c-484f-8092-c94d40a9b533)

Best regards,
Maciej Błędkowski

**Comment by angelhdzmultimedia** [comment]
@nuxtjs/i18n uses most of these formats, except .toml... :(

![image](https://github.com/nuxt/nuxt/assets/10191085/d82ea962-08bd-472a-9853-2bcb6825689d)

You can use a node package that transform .toml to object, but all I've searched are not ready for ESM, so you would have to use `require` with all the issues that comes with.
